packages/bioformats/superbuild.cmake

```diff
@@ -28,20 +28,20 @@ if(bf-dir)
     DOWNLOAD_COMMAND "")
   set(EP_SOURCE_DIR "${bf-dir}")
   set(BOOST_VERSION 1.60)
-  message(STATUS "Building Bio-Formats from local directory (${bf-dir})")
+  message(STATUS "Building OME XML C++ from local directory (${bf-dir})")
```

Comment:
It's updating the message to reflect that ome-xml is the only component remaining here now the rest is split out.